### DISCUSSION TITLE
可以给服务器本机代理吗

### DISCUSSION THREAD

**Discussion by z1073** [v2rayA/v2rayA#1130] [Q&A]
局域网其他机器连接没问题，本机翻不出去，如何让本机也能一起访问外网呢？系统环境 ubuntu server 22.04
![image](https://github.com/v2rayA/v2rayA/assets/50789523/f33980a2-a30e-4347-af44-88cd3d99fec9)

**Comment by zuilintan** [comment]
我这边可以, 
openwrt的docker环境, host网络模式, 
![image](https://github.com/v2rayA/v2rayA/assets/34953840/d2aa4b24-a1c5-49bf-afb4-7440f6c36e90)
![image](https://github.com/v2rayA/v2rayA/assets/34953840/728bdf87-e166-4913-a026-2b637261ab9d)

**Comment by hotinewang** [comment]
我想知道怎么共享给其他设备，比方说手机使用？如何设置？例如装V2raya的NASIP是http://192.168.1.29:2017/手机上该怎么设置代理？
